ApplyToBecomeInternal/ApplyToBecome.Data/Models/AdvisoryBoardDecision/DecisionMadeBy.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System.ComponentModel;
+
+namespace ApplyToBecome.Data.Models.AdvisoryBoardDecision
+{
+	public enum DecisionMadeBy
+	{
+		[Description("Regional Director for the region")]
+		RegionalDirectorForRegion = 0,
+		[Description("A different Regional Director")]
+		OtherRegionalDirector = 1,
+		[Description("Minister")]
+		Minister = 2,
+		[Description("Director General")]
+		DirectorGeneral = 3,
+		[Description("Other")]
```

Comment:
Enumeration name and description do not match. Description should be "None", right?

Reply:
I took this model from the API, but in the UI the label is set to Other. So I would need to change the name to match the description. Which I can certainly do. @MartinWheelerMT  what do you think?

Reply:
On the prototype the field label is None, and it's None in the model, so I think we should just stick with None

Reply:
hmm, maybe Iooked at sprint 35 prototype. Anyway will change to None.

Reply:
This has been resolved in the latest commit.